src/main/java/com/example/decafe/Machine.java

```diff
@@ -108,84 +135,102 @@ public void doProgressBarAnimation(Timer productionTimer, ImageView machineImage
                 new TimerTask() {
                     @Override
                     public void run() {
+
                         // After a certain time was reached change the Image of the Machine
                         machineImageView.setImage(imageProductProduced);
+
                         // And make the Machine clickable again
                         machineImageView.setDisable(false);
+
                         // And stop all timers
                         task.stop();
                         timelineBar.stop();
                         productionTimer.cancel();
                     }
                 },
-                this.duration* 1000L
+                this.duration * 1000L
         );
     }
 
     // Method to display a Product / change the state of a Machine
```

Comment:
The function is too ugly try to improve it, https://www.baeldung.com/java-replace-if-statements

Reply:
I agree, however let's not go overboard